app/resources/admin.html.template

```diff
@@ -42,7 +42,11 @@
 
 //TODO: move to object-oriented!!
 
-  var LANGUAGE_EXONYMS = {{language_exonyms_json|safe}};
+  var LANGUAGES = JSON.parse(
```

Comment:
(optional) It may be less error-prone to use a single variable to pass json data from the controller to the view. e.g. view_config_json, language_exonyms_json and everything else packed in json_data.

var JSON_DATA = JSON.parse("{{json_data|escapejs}}");